src/main/resources/workflows/0.3.0/irida_workflow.xml

```diff
@@ -39,7 +39,16 @@
     <parameter name="shovill-1-assembler" defaultValue="spades">
       <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/iuc/shovill/shovill/1.1.0+galaxy1" parameterName="assembler"/>
     </parameter>
-    <parameter name="staramr-2-use_pointfinder" defaultValue="disabled">
+    <parameter name="staramr-2-use_pointfinder" required="true">
+      <choices>
+	<choice name="disabled" value="disabled"/>
+	<choice name="campylobacter" value="campylobacter"/>
+        <choice name="enterococcus_faecalis" value="enterococcus_faecalis"/>
+        <choice name="enterococcus_faecium" value="enterococcus_faecium"/>
+        <choice name="escherichia_coli" value="escherichia_coli"/>
+        <choice name="helicobacter_pylori" value="helicobacter_pylori"/>
+        <choice name="salmonella" value="salmonella"/>
+      </choices>
       <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.9.1+galaxy0" parameterName="use_pointfinder"/>
```

Comment:
I think the `parameterName` here needs to be set to `pointfinder_db.use_pointfinder` in order to match the conditional name in the Galaxy tool

https://github.com/phac-nml/galaxy_tools/blob/c32ec4a9284ec939d001bd7d7d551c15feaa5333/tools/staramr/staramr_search.xml#L83

Reply:
Good catch! Fixed.